themes/sensei-course-theme/single.php

```diff
@@ -40,6 +40,11 @@
 	<div class="wp-block-column sensei-course-theme__sidebar sensei-course-theme__frame" style="flex-basis:300px">
 		<!-- wp:sensei-lms/focus-mode-toggle /-->
 		<!-- wp:sensei-lms/course-navigation /-->
+
+		<!-- wp:sensei-lms/button-contact-teacher -->
+		<a class="sensei-course-theme-contact-teacher__button">Contact Teacher</a>
```

Comment:
Just noting that it won't be translatable like this, so if we are not putting it a widget area, we'll need to update it, and maybe include this part in the render function if it has an empty content.

Reply:
Nice catch! I didn't think of that. I included it in the render function if it has an empty content. dcff7bb3b5d3746e886faf4f7fd53b28360d4814